en/index.md

```diff
@@ -4,12 +4,16 @@ Welcome to the __Cocos Creator__ User Manual! __Cocos Creator__ is a free, cross
 
 **Attention**: Please select the corresponding version of the manual in the upper right corner according to the version of Cocos Creator you are using. The current stable version is still v2.0, and if you are using v2.1, please refer to the [Cocos Creator v2.1 User Manual](../../2.1/manual/en/index.html).
 
-## What's New
 
- - Starting with v1.10, Cocos Creator has refactored the underlying resource types, most of which are unaffected, but some projects may receive warnings, please refer to [v1.10 Resource Upgrade Guide](release-notes/raw-asset-migration.md).
- - Starting with v1.9, Cocos Creator support posted to [Facebook Instant Games](publish/publish-fb-instant-games.md) and [QQ Play](publish/publish-qqplay.md).
- - Starting with v1.8，Cocos Creator support posted to Wechat Mini Games，please refer to [Wechat Mini Games](publish/publish-wechatgame.md).
- - Cocos Creator also supports exporting scenes and UI to Cocos2d-x, see [C++/Lua Support](advanced-topics/cpp-lua.md).
+## What's New
```

Comment:
Can you help delete the following three jump links? Including the zh-docs

![image](https://user-images.githubusercontent.com/19702899/49507592-bee43600-f8bb-11e8-8524-f8f229a64c37.png)